src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java

```diff
@@ -124,6 +143,23 @@ void count() {
 				airportRepository.save(airport).block();
 			}
 
+			int page = 0;
+			for (; page < 10;) {
+				PageRequest pageable = PageRequest.of(page++, 2);
+				System.out.println("findAllByIataLike: " + "%S");
+				// java.lang.IndexOutOfBoundsException: Source emitted more than one item
+				// Flux<Airport> airportFlux = airportRepository.findAllByIataLike("S%");
```

Comment:
can this be removed or does this needs to be handled?

Reply:
I changed this to actually use the paging. Removed the commented-out lines and added assertions.